src/Polly.Core/Simmy/MonkeyStrategy.TResult.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace Polly.Simmy;
+
+#pragma warning disable RS0027 // API with optional parameter(s) should have the most parameters amongst its public overloads
+
+/// <summary>
+/// Contains common functionality for chaos strategies which intentionally disrupt executions - which monkey around with calls.
+/// </summary>
+/// <typeparam name="T">The type of result this strategy supports.</typeparam>
+/// <remarks>
+/// Resilience strategy supports various types of callbacks of <typeparamref name="T"/> result type
+/// and provides a unified way to execute them. This includes overloads for synchronous and asynchronous callbacks.
+/// </remarks>
+public class MonkeyStrategy<T>
```

Comment:
You can delete this class as generic strategies are transparently handled by Polly.

If you need a base class for outcome (reactive) strategies you can use `ReactiveMonkeyStrategy`.